### ISSUE TITLE
Publish the app as a .NET Core Global Tool

### ISSUE THREAD

**Issue by petrsvihlik** [Kentico/cloud-generators-net#73]
### Motivation

> A .NET Core Global Tool is a special NuGet package that contains a console application. A Global Tool can be installed on your machine on a default location that is included in the PATH environment variable or on a custom location.

Having the generator installed as a global tool will allow for re-generating models as per https://github.com/Kentico/cloud-boilerplate-net/issues/31 .

### Design guidelines

The goal of this task is to make changes necessary to publish the Kentico Cloud Code Generator for .NET as a .NET Core Global Tool via NuGet.

### Resources

- [Official documentation](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools)
- [Howto blogpost](https://natemcmaster.com/blog/2018/05/12/dotnet-global-tools/)
- [Example](https://www.contentful.com/blog/2018/06/27/tool-creating-classes-content-model/)

**Comment by matus12** [comment]
I have looked at this and came across an issue while packing the NuGet package: _The "GetRuntimeLibraries" task was not given a value for the required parameter "RuntimeIdentifier"._ I have tried setting the runtime argument for _dotnet pack_ and also setting runtime identifiers to _csproj_, but it didn't help. The problem is related to the _ILLink.Tasks_ package that we use, I have tried other versions of the package - didn't work, the only working solution I found to this is to remove the package. I have also searched for alternatives to _ILLink.Tasks_ but found only tools that have to be run manually.
